src/main/java/org/jabref/gui/JabRefFrame.java

```diff
@@ -429,8 +429,8 @@ private void initLayout() {
         head.setSpacing(0d);
         setTop(head);
 
-        splitPane.getItems().addAll(sidePane, tabbedPane);
-        SplitPane.setResizableWithParent(sidePane, false);
+        splitPane.getItems().addAll(sidePaneContainerView, tabbedPane);
+        SplitPane.setResizableWithParent(sidePaneContainerView, false);
```

Comment:
Personally im not very happy with the name `SidePaneContainerView`. It may be technically correct, but we also don't use MainTableView or CheckMenuItemView. `SidePane` was easier to read...
But it's probably just a matter of definitions: According to you the objects formerly called components in this container are the real sidepanes, and this is just an empty container, that needs to be filled...

Reply:
I understand your point, indeed SidePane is the real SidePane and components are an internal detail that we can manage/access through methods provided by SidePane.
If we rename `SidePaneView` to `SidePaneComponent` then we need to also rename `SidePaneType` to `SidePaneComponentType` or `SidePaneItem`or create a subclass in `SidePaneComponent` called `Type`, `SidePaneComponent.Type`, I would like to hear your thoughts about this.

Reply:
Sounds good. But again, this is just my personal opinion.